core/domain/user_services.py

```diff
@@ -1971,29 +1969,29 @@ def remove_community_reviewer(user_id):
         user_community_rights_model.delete()
 
 
-def get_community_reviewer_usernames(review_type, language_code=None):
+def get_community_reviewer_usernames(review_category, language_code=None):
     """Returns a list of usernames of users who has rights to review item of
-    given review type.
+    given review category.
 
     Args:
-        review_type: str. The type of of review item. One of translation,
-            voiceover or question.
+        review_category: str. str. The category which for in which user can
```

Comment:
The review category to find the list of reviewers for.

Reply:
Done!